docs/dev-setup.md

```diff
@@ -123,13 +123,31 @@ After you have cloned both repositories, your folder structure should look like:
 |-- src/cluster-api (run `tilt up` here)
 ```
 
+Checkout a stable CAPI release, for example:
+
+```bash
+git checkout release-0.3
+```
+
+Install kustomize from within the cluster-api repo
+
+```bash
+make kustomize
+
+# verify installation
+./hack/tools/bin/kustomize version
+```
+
+If the above installation does not work, install kustomize using a package manager and move the binary to ```hack/tools/bin/kustomize``` in the cluster-api repo.
+
 After configuring the environment variables, run the following to generate your
 `tilt-settings.json` file in the cluster API repository:
 
 ```shell
 cat <<EOF > tilt-settings.json
 {
     "allowed_contexts": ["kind-capm3"],
+    "default_registry": "gcr.io/cluster-api-provider",
```

Comment:
Why is this needed @Xenwar ?

Reply:
the command that uses tis configuration file requires it. It was needed before